docs/source/reference/yaml-spec.rst

```diff
@@ -153,6 +156,18 @@ Available fields:
     #
     # Values set here can be overridden by a CLI flag:
     # `sky launch/exec --env ENV=val` (if ENV is present).
+    #
+    # If you want to use a docker image in a private registry, you can specify your
+    # username, password, and registry server as task environment variable. For example:
+    #   envs:
+    #     SKYPILOT_DOCKER_USERNAME: <username>
+    #     SKYPILOT_DOCKER_PASSWORD: <password>
+    #     SKYPILOT_DOCKER_SERVER: <registry server>
+    # SkyPilot will execute `docker login --username <username> --password <password> <registry server>`
+    # before pulling the docker image.
```

Comment:
Add a link to the docker doc?

Reply:
Seems like RST doesn't support define an label or reference to other places in a code block since it will be treated as part of code... I add a plain text reference now

Reply:
Yes, I was trying to say adding a url in plain text : )